tools/crm_mon.c

```diff
@@ -696,13 +702,29 @@ main(int argc, char **argv)
         }
     }
 
-    if (optind < argc) {
+    /* Extra sanity check when in CGI mode */
+    if (output_format != mon_output_cgi) {
```

Comment:
Thanks for catching, burnt by reordering patches and somehow reintroducing the thinko, I've been there already :-/